Automattic/wp-calypso
# [eslint] Fix jest/no-test-callback

#### Changes proposed in this Pull Request

* Apply automatic fix for rule `jest/no-test-callback`

[Rule doc](https://github.com/jest-community/eslint-plugin-jest/blob/v23.6.0/docs/rules/no-test-callback.md)

#### Testing instructions

* Verify unit tests still pass. Ignore lint tests.
* Run ./node_modules/.bin/eslint-nibble --ext .js,.jsx,.ts,.tsx . in this branch and check there are no more violations of `jest/no-test-callback` (warning: it takes several minutes)

In master:
![image](https://user-images.githubusercontent.com/975703/94643447-1badad80-02e7-11eb-9a87-8cf3b14cc9b1.png)

In this branch:
![image](https://user-images.githubusercontent.com/975703/94657325-95529500-0301-11eb-95f7-1b572c4ed68c.png)
